src/bin/pango_attributes.rs

```diff
@@ -0,0 +1,56 @@
+//! # Pango text attributes
+//!
+//! This sample demonstrates how to use various attributes (strikeout, background color, etc. ) on labels text
```

Comment:
Missing dot at the end of the line. Also, can you keep the line's length under 100 column please?